### PR TITLE
[thci] add SSH keepalive

### PR DESCRIPTION
**Pull request by jcdong98**
Add SSH keepalive to avoid disconnection after idle for a long time.

### CODE REVIEW

File: tools/harness-thci/OpenThread_BR.py [hunk 1]
```diff
@@ -60,6 +60,7 @@
 
 
 class SSHHandle(object):
+    KEEPALIVE_INTERVAL = 30
```
Comment (superwhd) [inline tools/harness-thci/OpenThread_BR.py:63]: nit: specify the unit in comment.

**Review by superwhd — APPROVED** [review]
LGTM :+1:

File: tools/harness-thci/OpenThread_BR.py (hunk 1 above)
Comment (jcdong98) [reply tools/harness-thci/OpenThread_BR.py:64]: Added the comment.

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍

**Review by jwhui — APPROVED** [review]

**Pull request merged**